### PR TITLE
CRIT Expansion Update 

### PR DESCRIPTION
**Pull request by Soupster89**
#### Summary
```SUMMARY: Mods "Update"```

#### Purpose of change

The start of a line of changes that aims to slowly flesh out the identity of the _CRIT_ content mod. My mission is to refine a mod that evolves into a more polished experience that reflects well on the devs and maintainers that are a part of this wonderful community. 

#### Basic rebalance or additions:

- MA Styles have been balanced to fit their respective themes.

- The Rivet Gun no longer produces explosions or shrapnel.

- Guns have been majorly buffed

- Techniques offer less eye-stress in regards to the descriptions and should feel more "correct"

- Minor Mutation additions

- Various items have been changed

#### Major Changes:

- Dark Souls content will become a new mod PR (moved for now)

- Dead Space content will stay to avoid destroying saves

- Extended Techniques will stay in the mod (to avoid destroying old saves)

#### Checklist

Resolve all the issues and get back into standards:

- [x] Weight Strings

- [x] Sufficient Spaces

- [x] Miscellaneous

Re-do files that have been modified to pass conflicts back to rebalanced version:

- [x] Items

- [x] Constructions

- [x] Martial arts

- [x] Classes/ Scenarios

### CODE REVIEW

**Comment by Soupster89** [comment]
Having difficulty seeing if enemies spawn or not, unsure if its because its RNG or because it's being overlooked.

**Comment by Moonsabrt** [comment]
Stumbled on this looks cool famalam!

**Comment by ZhilkinSerg** [comment]
`"repaired_with": [ "plastic_chunk", "chunk_rubber" ],`

should be

`"repaired_with": "chunk_rubber",`

**Comment by Soupster89** [comment]
Sound and tiles build not passing?

**Comment by Soupster89** [comment]
Another check in the box!

**Comment by Soupster89** [comment]
I'm going to get started on this fouling thing.

**Comment by Soupster89** [comment]
It just needs playtesting!!!!

**Comment by Soupster89** [comment]
Doesn't run correctly no matter what I do?

**Comment by Maleclypse** [comment]
> Doesn't run correctly no matter what I do?

I saw the same "sap" and "poison" errors that seem to be coming up alot lately but nothing related directly to your changes.  If you're getting no errors when you playtest it I would say you can probably remove the WIP and hopefully the tests will stop ringing false positives soon.

**Comment by Soupster89** [comment]
> > Doesn't run correctly no matter what I do?
> 
> I saw the same "sap" and "poison" errors that seem to be coming up alot lately but nothing related directly to your changes. If you're getting no errors when you playtest it I would say you can probably remove the WIP and hopefully the tests will stop ringing false positives soon.

Thank you, I have resolved the issue and it now works just fine and dandy.

**Comment by Moonsabrt** [comment]
Your working on this again?
'w') If so glad to hear it :D

**Comment by Soupster89** [comment]
Hello, how do I fix this?

File: data/mods/CRT_EXPANSION/items/crt_ammo.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "color": "light_gray",
     "description": "A round tin of small light grain .177 lead pellets.  These are common, tipped field pellets that can deal some light damage but are generally used for plinking.",
     "material": "steel",
-    "volume": "200ml",
+    "volume": 0.4,
```
Comment (LaVeyanFiend) [inline data/mods/CRT_EXPANSION/items/crt_ammo.json]: Change back to metric string, please.

File: data/mods/CRT_EXPANSION/items/crt_ammo.json [hunk 2]
```diff
@@ -62,7 +62,7 @@
     "color": "light_blue",
     "description": "A helical magazine of hollow-point alloy bullets propelled by pockets of primer. Not the most lethal thing out there, but it still packs a punch without the worry of having a stray shot seriously damaging the environment.",
     "material": [ "steel", "powder" ],
-    "volume": "250ml",
+    "volume": 0.5,
```
Comment (LaVeyanFiend) [inline data/mods/CRT_EXPANSION/items/crt_ammo.json]: ^

File: data/mods/CRT_EXPANSION/items/crt_toolarmor.json [hunk 3]
```diff
@@ -220,10 +217,34 @@
     "use_action": "HOTPLATE"
   },
   {
-    "id": "bandana_wet",
+    "id": "packaged_bandana_wet",
```
Comment (LaVeyanFiend) [inline data/mods/CRT_EXPANSION/items/crt_toolarmor.json]: You'd have to obsolete the old ID and migrate to this one, or it breaks.

File: data/mods/CRT_EXPANSION/items/crt_toolarmor.json [hunk 4]
```diff
@@ -220,10 +217,34 @@
...
+      "msg": "You unpack your wet bandana",
+      "target": "unpacked_bandana_wet",
+      "active": true,
+      "need_charges": 60,
+      "need_charges_msg": "Seeing this is a bug"
+    },
+    "flags": [ "WET" ]
+  },
+  {
+    "id": "unpacked_bandana_wet",
```
Comment (LaVeyanFiend) [inline data/mods/CRT_EXPANSION/items/crt_toolarmor.json]: ^

File: data/mods/CRT_EXPANSION/items/crt_toolarmor.json (hunk 3 above)
Comment (Soupster89) [reply data/mods/CRT_EXPANSION/items/crt_toolarmor.json]: Do I remove it?

Comment (LaVeyanFiend) [reply data/mods/CRT_EXPANSION/items/crt_toolarmor.json]: 2. Create a migration.json file inside the items folder, add this to it:
```
{
    "id": "bandana_wet",
    "type": "MIGRATION",
    "replace": "packaged_bandana_wet"
  }
```
3. If there's an associated recipe already, add an obsolete.json to *recipes* folder and add this:
```
{
    "type": "recipe",
    "result": "bandana_wet",
    "obsolete": true
  }
```

Comment (Soupster89) [reply data/mods/CRT_EXPANSION/items/crt_toolarmor.json]: Oh ok, nice!

**Comment by Maddremor** [comment]
Is there any coherent theme of this mod? Plant fiber clothing, Dark Souls, and sci-fi tactical stuff? 
This seems to clash with the "Content mods need to be providing some kind of curated experience" clause of the [in-repo mod inclusion rules](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/IN_REPO_MODS.md).

**Comment by Soupster89** [comment]
I see.

**Comment by Soupster89** [comment]
> Is there any coherent theme of this mod? Plant fiber clothing, Dark Souls, and sci-fi tactical stuff?
> This seems to clash with the "Content mods need to be providing some kind of curated experience" clause of the [in-repo mod inclusion rules](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/IN_REPO_MODS.md).

Correct! I will fix these issues the best as I can. : )

**Comment by Soupster89** [comment]
Just needs more thorough playtesting.

**Comment by Soupster89** [comment]
Nice! I did it guys!

I have stamped out all the bugs and now I can definitely run a 100% good play test.

**Comment by Maleclypse** [comment]
> Nice! I did it guys!
> 
> I have stamped out all the bugs and now I can definitely run a 100% good playtest.
> 
> Edit: Nvm, something else went wrong.

Hate when that happens but it at least it's progress. :)

File: data/mods/CRT_EXPANSION/mutations/crt_wendigo_mutations.json [hunk 5]
```diff
@@ -99,5 +99,43 @@
...
+    "valid": false,
+    "purifiable": false,
+    "threshreq": [ "THRESH_WENDIGO" ],
+    "category": [ "WENDIGO" ],
+    "social_modifiers": { "intimidate": 5 },
+    "attacks": [
+      {
+        "attack_text_u": "You ravage %s with your arms",
+        "attack_text_npc": "%1$s ravages %2$s with their arm spurs",
+        "body_part": "arm_r",
```
Comment (Wishbringer) [inline data/mods/CRT_EXPANSION/mutations/crt_wendigo_mutations.json:120]:
```suggestion
        "body_part": "ARM_R",
```

**Comment by Soupster89** [comment]
I am finally ready, whoo hoo

**Pull request merged**